test/oidc-strategy.ts

```diff
@@ -0,0 +1,250 @@
+import * as Lab from "@hapi/lab";
+import { expect } from "@hapi/code";
+import { stub } from "sinon";
+import { Server } from "@hapi/hapi";
+
+// @ts-ignore dumb config stuff makes me ts-ignore json files
```

Comment:
Is this necessary? Removing it didn't throw any statics analysis or compilation errors in my environment.
```suggestion
```